nessie/src/test/java/org/apache/iceberg/nessie/TestNessieTable.java

```diff
@@ -0,0 +1,349 @@
+  @Test
+  public void testRename() {
+    String renamedTableName = "rename_table_name";
+    TableIdentifier renameTableIdentifier = TableIdentifier.of(TABLE_IDENTIFIER.namespace(),
+                                                               renamedTableName);
+
+    Table original = catalog.loadTable(TABLE_IDENTIFIER);
+
+    catalog.renameTable(TABLE_IDENTIFIER, renameTableIdentifier);
+    Assert.assertFalse(catalog.tableExists(TABLE_IDENTIFIER));
+    Assert.assertTrue(catalog.tableExists(renameTableIdentifier));
+
+    Table renamed = catalog.loadTable(renameTableIdentifier);
+
+    Assert.assertEquals(original.schema().asStruct(), renamed.schema().asStruct());
+    Assert.assertEquals(original.spec(), renamed.spec());
+    Assert.assertEquals(original.location(), renamed.location());
+    Assert.assertEquals(original.currentSnapshot(), renamed.currentSnapshot());
+
+    Assert.assertTrue(catalog.dropTable(renameTableIdentifier));
+  }
+
+  @Test
+  public void testDrop() {
+    Assert.assertTrue(catalog.tableExists(TABLE_IDENTIFIER));
+    Assert.assertTrue(catalog.dropTable(TABLE_IDENTIFIER));
+    Assert.assertFalse(catalog.tableExists(TABLE_IDENTIFIER));
+  }
+
+  @Test
+  public void testDropWithoutPurgeLeavesTableData() throws IOException {
+    Table table = catalog.loadTable(TABLE_IDENTIFIER);
+
+    GenericRecordBuilder recordBuilder =
+        new GenericRecordBuilder(AvroSchemaUtil.convert(schema, "test"));
+    List<GenericData.Record> records = new ArrayList<>();
+    records.add(recordBuilder.set("id", 1L).build());
+    records.add(recordBuilder.set("id", 2L).build());
+    records.add(recordBuilder.set("id", 3L).build());
+
+    String fileLocation = table.location().replace("file:", "") + "/data/file.avro";
+    try (FileAppender<GenericData.Record> writer = Avro.write(Files.localOutput(fileLocation))
+                                                       .schema(schema)
+                                                       .named("test")
+                                                       .build()) {
+      for (GenericData.Record rec : records) {
+        writer.add(rec);
+      }
+    }
+
+    DataFile file = DataFiles.builder(table.spec())
+                             .withRecordCount(3)
+                             .withPath(fileLocation)
+                             .withFileSizeInBytes(Files.localInput(fileLocation).getLength())
+                             .build();
+
+    table.newAppend().appendFile(file).commit();
+
+    String manifestListLocation =
+        table.currentSnapshot().manifestListLocation().replace("file:", "");
+
+    Assert.assertTrue(catalog.dropTable(TABLE_IDENTIFIER, false));
+    Assert.assertFalse(catalog.tableExists(TABLE_IDENTIFIER));
+
+    Assert.assertTrue(new File(fileLocation).exists());
+    Assert.assertTrue(new File(manifestListLocation).exists());
+  }
+
+
+  @SuppressWarnings("VariableDeclarationUsageDistance")
```

Comment:
What is this suppressing?

Reply:
absolutely nothing! It got carried over from the nessie repo (which has militant checkstyle rules) and I forgot to remove it. Gone now.